app/controllers/shop_controller.rb

```diff
@@ -5,6 +5,8 @@ class ShopController < BaseController
   before_filter :require_distributor_chosen
   before_filter :set_order_cycles
 
+  #after_filter :enable_embedded_shopfront
+
```

Comment:
I've changed this line now, but yes it's just the shop and shops controllers that can be directly accessed in an iframe. The way they work is that once you've accessed one page in an iframe initially, you can then navigate to any other page within the iframe.